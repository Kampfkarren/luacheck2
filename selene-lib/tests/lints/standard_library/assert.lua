assert(true, "message")
assert(call())
assert(true)
assert(call(), "this is ok")
