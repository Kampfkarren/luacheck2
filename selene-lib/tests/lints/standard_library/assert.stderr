error[incorrect_standard_library_use]: standard library function `assert` requires 2 parameters, 1 passed

   ┌── assert.lua:3:1 ───
   │
 3 │ assert(true)
   │ ^^^^^^^^^^^^
   │

